virtcontainers/qemu.go

```diff
@@ -565,19 +565,19 @@ func (q *qemu) qmpSetup() (*govmmQemu.QMP, error) {
 	return qmp, nil
 }
 
-func (q *qemu) addDeviceToBridge(ID string) (string, string, error) {
+func (q *qemu) addDeviceToBridge(ID string) (string, *Bridge, error) {
```

Comment:
Yes,  copy should be enough, I tend to pass addresses to avoid extra copies. I believe any incorrect changes should be caught by our unit tests/CI.
I can change this though, would like more input on this.

Reply:
so.. , a copy should be enough? or no?

Reply:
Like I mentioned copy should be enough. I tend to use addresses to avoid copies and create empty objects in case of error. I'll change this if this is a blocker.

Reply:
@devimc Fixed this.